Gemfile.lock

```diff
@@ -371,3 +371,4 @@ DEPENDENCIES
 
 BUNDLED WITH
    1.10.6
+   
```

Comment:
@swaathi there's still changes in the Gemfile.lock which shouldn't be there. It seems there's whitespace on the last line, I think if you remove that we're golden :ok_hand: 

(Remember to keep your commits squashed)